### PR TITLE
Adjust colour for warning message

### PR DESCRIPTION
**Pull request by flamerohr**
Follow up for https://github.com/silverstripe/silverstripe-versioned/issues/71 (got a more accurate colour)

### CODE REVIEW

File: client/src/components/FormAction/FormAction.scss [hunk 1]
```diff
@@ -306,7 +306,7 @@ button.close {
 }
 
 .actions-warning {
-  color: darken($state-modified-bg, 15);
+  color: #856404; // in bootstrap: theme-color-level(map-get($theme-colors, "yellow"), 6)
```
Comment (robbieaverill) [inline client/src/components/FormAction/FormAction.scss]: Are we able to use a variable, out of interest?

**Comment by tractorcow** [comment]
I'm not sure if I find the former or the latter colour more repulsive, but we'll follow the designs so we can't say we didn't do as we were told. :)

**Comment by tractorcow** [comment]
Merge it on green.

File: client/src/components/FormAction/FormAction.scss (hunk 1 above)
Comment (flamerohr) [reply client/src/components/FormAction/FormAction.scss]: I had opted out of using a variable for now, if we're finding ourselves using more of bootstrap's mixin'd theme colours we can setup a block in the _variables.scss file

**Pull request merged**